src/paths.ts

```diff
@@ -74,12 +146,23 @@ class Paths {
    */
   get roamingDataDir(): string { return path.join(this.userRoamingDir, 'CMakeTools'); }
 
+  /**
+   * Returns e.g. "C:\" on Windows and "/" on Unix-like systems
+   */
+  get rootPath(): string {
```

Comment:
It didn't look like this was used anywhere?  Can you remove it?

Reply:
My intention is to export this as a substitution variable `${rootPath} to be used in `settings.json` or wherever it might be needed. I haven't done that yet. For example a default path to MinGW is set in package.json as `C:\\MinGW`. I actually did it but then removed it since I wasn't able to test it.

Reply:
If you have a plan to use it then it's ok to leave it in. 😄